krizten/Memoire-WebClient
# #165530505 view profile page

### What does this PR do?
- Build out the profile page
#### Description of Task to be completed?
- Develop markup using provided UI/UX design.
#### How should this be manually tested?
- Clone this repository and checkout to the branch: ```ft/165530505/view-profile-page```
- Ensure [yarn](https://yarnpkg.com/lang/en/) is installed, then install the project dependencies by simply running ```yarn``` in the terminal.
- Start the project using ```yarn start```.
- Using a web browser, navigate to ```http://localhost:3000/profile```
#### What are the relevant pivotal tracker stories?
#165530505
#### Screenshots (if appropriate)

![profile-desktop](https://user-images.githubusercontent.com/29978455/58507082-cc9eb800-8188-11e9-9bc3-fc59d0e3062e.png)
![profile-mobile](https://user-images.githubusercontent.com/29978455/58507098-d32d2f80-8188-11e9-8000-077e5e0be5e9.png)
